### DISCUSSION TITLE
PHPMailer working....loading... mod_php

### DISCUSSION THREAD

**Discussion by petronioamaral** [PHPMailer/PHPMailer#2362] [General]
Hi there,

Someone had problem when using mod_php ( php 7.2 ) ?
When I clicked send, my code has working loading a longtime and not send.

Thanks

**Comment by Synchro** [comment]
mod_php is fine. It's most likely that your hosting provider is blocking outbound SMTP and your script is simply waiting for the usual SMTP connection timeout, which is 10 minutes. Please read the troubleshooting guide which tells you all this and shows you how to diagnose it.
